product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -205,6 +205,14 @@ Enable **Identity and Access Management (IAM) Authentication** to turn on the ab
 
 Enable **Superuser Access** to grant superuser privileges to the edb_admin role. This option is available for single-node and primary/standby high-availability clusters. See [Notes on the edb_admin role](/biganimal/latest/using_cluster/01_postgres_access/#notes-on-the-edb_admin-role).
 
+### Security 
+
+Enable **Transparent Data Encryption (TDE)** to use your own encryption key. This option is available for EDB Postgres Advanced Server and on AWS. Select an encryption key from your project and region to encrypt the cluster with TDE.
+
+!!!Note
+To enable and use TDE for cluster, the encryption key must be enabled and added at the project level. For more information, see [Adding a TDE key at project level](../../administering_cluster/projects.mdx/#adding-a-tde-key).
+!!!
+
```

Comment:
also, a new cluster won't reach healthy state until the user grants BA permissions to their key in their cloud account. see this screen here: https://www.figma.com/file/XZPReYBzieivob6PykaLKb/Transparent-Data-Encryption-in-BigAnimal?type=design&node-id=336-49094&mode=design&t=b4PN2SgvYrf73sbE-0

Reply:
@nidhibhammar We should also add Principal information that is being displayed in Overview page of the PG cluster so user can copy that principal and add in their key policy to give `kms:Encrypt` and `kms:Decrypt` permission so it will be easy for end user.

cc: @amrita-suresh @pramod-ahire 

e.g.

```
.......
           "Principal": {
               "AWS": "<Managed_identity_Of_PG_cluster>" /* this will be displayed to user on Overview page that user have to replaced it here... */
           },
           "Action": [
               "kms:Encrypt",
               "kms:Decrypt"
           ],
.......
```